templates/httpd.conf.erb

```diff
@@ -109,6 +109,9 @@ LogFormat "<%= format -%>" <%= nickname %>
   <%- end -%>
 <% end -%>
 
+<%- if scope.function_versioncmp([@apache_version, '2.4']) >= 0 -%>
+IncludeOptional "<%= @conf_dir %>conf-enabled/*.conf"
```

Comment:
Missing at least a slash. This will result in `/etc/apache2conf-enabled/*.conf`.

It's also rather ugly to do this on all distros, even those that don't have this mechanism. I wonder if we need a `$conf_enable_dir` that's `undef` on all other distros so you can test for it in the template.

Reply:
@ekohl Have updated the code to be along the lines of what you suggested